test/hotspot/gtest/aarch64/aarch64-asmtest.py

```diff
@@ -1522,6 +1522,7 @@ def generate(kind, names):
                         ["stxp",   "__ stxp(r4, zr, zr, r5);",                           "stxp\tw4, xzr, xzr, [x5]"],
                         ["stxpw",  "__ stxpw(r6, zr, zr, sp);",                          "stxp\tw6, wzr, wzr, [sp]"],
                         ["dup",    "__ dup(v0, __ T16B, zr);",                           "dup\tv0.16b, wzr"],
+                        ["dups",   "__ dups(v0, __ T2S, v1);",                           "mov\ts0, v1.s[0]"],
```

Comment:
This equivalence would be easier to understand as
`                         ["dups",   "__ dups(v0, __ T2S, v1);",                           "dup\ts0, v1.s[0]"],`

Reply:
Thanks, I'll fix it.